dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/BasicDeploymentForm.test.tsx

```diff
@@ -131,16 +131,25 @@ it("should hide an element if it depends on a param (object)", () => {
       path: "foo",
       type: "string",
       hidden: {
-        value: "bar",
-        condition: "enabled",
+        conditions: [
+          {
+            value: "bar",
+            reference: "enabled",
```

Comment:
nit: I would rename `reference` with `path` to make it more clear that is a jsonpath